src/components/ProofCard.vue

```diff
@@ -32,3 +50,20 @@ export default {
   }
 }
 </script>
+
+<style scoped>
+.highlighted {
```

Comment:
never used ?

Reply:
Not with the current implementation of showing the card in fullscreen when selected. However if I remove that implementation, do we want to highlight or just show the confirmation dialog?

Reply:
neither ? proof simply selected on click

Reply:
Alright. I commited the changes